stage1/tasks/virtual-music-kit/README.md

```diff
@@ -0,0 +1,147 @@
+# Virtual Music Kit
+
+## Task
+
+### Description
+
+The Virtual Music Kit is an interactive sound application inspired by real musical instruments such as a piano, drums, or **any other instrument of your choice**.
+Users should be able to play sounds live (by clicking visual elements and/or by pressing assigned keyboard keys), or reproduce a sequence of available sounds of their choice.
+
+
+### What you will learn
+
+- DOM ­manipulation
+- Event handling
+- Audio playback in the browser
+
+
+### UI requirements
+
+1. The application's visual layout represents your chosen instrument (for example, piano keys, drum pads, guitar strings, etc.).  
+   It is not required to reproduce the real-life appearance of the instrument. You can design it creatively, but it must be immediately clear to users which instrument the application simulates.
+
+2. The application includes at least **7 distinct sounds (or notes)** related to the chosen instrument.
+- Each sound and its corresponding visual element must be mapped to **one unique keyboard key** (for example: `A`, `S`, `D`, `F`, etc.).
+- Only **English letter keys** can be used for this purpose.
+- Multiple sounds may belong to the **same instrument type** (for example, in a drum kit: an open hi-hat, closed hi-hat, and pedal hi-hat count as three different sounds).
+- The assigned keys must be **visibly displayed** on the screen and **clearly associated** with the corresponding instrument (or part of it) and sound.
+- Each sound must be triggered **either by pressing** its assigned keyboard key **or by clicking** on the corresponding visual element.
+
+3. At the beginning of each session, the app **randomly assigns keys** to the available sounds.  
```

Comment:
I think this is not a very good solution, because the control scheme should initially be set to be convenient for the user. If the controls are randomized every time, the user will be confused.

Perhaps it would be better for students to preset certain keys for the tools? And give the user the ability to reassign these keys